### ISSUE TITLE
Notification sound repeating forever

### ISSUE THREAD

**Issue by amarantex64** [eneshecan/whatsapp-for-linux#284]
**Description**
In version 1.6.2 there is a bug with the notification sound repeating forever until you close the app.

**Environment**
 - Distro [Ubuntu, Fedora, OpenSUSE, Manjaro]
 - Package [flatpak]
 - Desktop Environment or Window Manager [GNOME, KDE, Xfce]

**Comment by Switch123456789** [comment]
Same here!
AUR / Arch / EndeavourOS / Gnome /  Pipewire / v 1.6.2

**Comment by Switch123456789** [comment]
Since it might be related to this issue: Is this expected behavior?

![Screenshot from 2023-04-14 00-34-37](https://user-images.githubusercontent.com/25269380/231897824-603a1b96-5d29-453d-9549-312e351a357e.png)

**Comment by PmicroTO** [comment]
Same issue here, Fedora [copr](https://copr.fedorainfracloud.org/coprs/nunodias/whatsapp-for-linux/) and OpenSUSE's.

**Comment by eneshecan** [comment]
Could you guys please try to see if the application prints anything when that happens? You can simply run it from the terminal.

**Comment by Switch123456789** [comment]
@eneshecan I am on it for a few days now but couldn't catch it happening yet (I don't use whatsapp much). I might do some testing with a friend if I have an opportunity.

**Comment by Switch123456789** [comment]
Hey, I just wanna add some additional infos that might be important.
1. I recently ended up with this after having the client running for a while, I guess it spawns a new one whenever I play audio-files (purple icon) that were sent or received in WAFL but it doesn't "despawn" it?

![Screenshot from 2023-04-18 01-16-15](https://user-images.githubusercontent.com/25269380/234671895-5989bce7-4a9a-46f8-bff9-f3c2fbb67cc0.png)

2. These are some logs I got right after I started WAFL, not sure if this means anything important.

```
[switch@DESKTOP ~]$ whatsapp-for-linux 
<13>Apr 26 16:43:39 whatsapp-for-linux[4016]: 
<13>Apr 26 16:43:39 whatsapp-for-linux[4016]: ** (whatsapp-for-linux:4012): WARNING **: 16:43:39.817: Error loading plugin: libhspell.so.0: cannot open shared object file: No such file or directory
<13>Apr 26 16:43:39 whatsapp-for-linux[4016]: 
<13>Apr 26 16:43:39 whatsapp-for-linux[4016]: 
<13>Apr 26 16:43:39 whatsapp-for-linux[4016]: ** (whatsapp-for-linux:4012): WARNING **: 16:43:39.817: Error loading plugin: libvoikko.so.1: cannot open shared object file: No such file or directory
<13>Apr 26 16:43:39 whatsapp-for-linux[4016]: 
<13>Apr 26 16:43:39 whatsapp-for-linux[4016]: 
<13>Apr 26 16:43:39 whatsapp-for-linux[4016]: ** (whatsapp-for-linux:4012): WARNING **: 16:43:39.817: Error loading plugin: libaspell.so.15: cannot open shared object file: No such file or directory
<13>Apr 26 16:43:39 whatsapp-for-linux[4016]: 
<13>Apr 26 16:43:39 whatsapp-for-linux[4016]: 
<13>Apr 26 16:43:39 whatsapp-for-linux[4016]: ** (whatsapp-for-linux:4012): WARNING **: 16:43:39.818: Error loading plugin: libnuspell.so.5: cannot open shared object file: No such file or directory
<13>Apr 26 16:43:39 whatsapp-for-linux[4016]: 
<13>Apr 26 16:43:39 whatsapp-for-linux[4016]: SettingMap: Key file does not have key “min-font-size” in group “web”, returning default value: 0
<13>Apr 26 16:43:39 whatsapp-for-linux[4016]: SettingMap: Key file does not have key “header-bar” in group “general”, returning default value: 1
Opening in existing browser session.
<13>Apr 26 16:43:39 whatsapp-for-linux[4016]: libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
```

3. While trying to reproduce the problem I noticed I don't get notification-sounds at all. Maybe it is because I muted the entry when it was stuck in a loop but now I can't see it anymore because it disappears after playing the notification-sound, so I don't have a chance to turn it back on? I don't know!

**Comment by berserkwarwolf** [comment]
Same here, on Arch. The only way to make it stop is to quit the app and reopen, until it does that again.

**Comment by eneshecan** [comment]
I did something with a86df29 that might solve the issue. Would be nice to keep an eye on it.

**Comment by spifory** [comment]
Seems to still be making the sound for me. I've installed the app from the [AUR](https://aur.archlinux.org/packages/whatsapp-for-linux), and I'm on version 1.6.3-1.

**Comment by eneshecan** [comment]
> Seems to still be making the sound for me. I've installed the app from the [AUR](https://aur.archlinux.org/packages/whatsapp-for-linux), and I'm on version 1.6.3-1.

Does it happen always?

**Comment by spifory** [comment]
Yep, as soon as someone sends me a message and the notification plays, it just continuously plays the sound until I re-open it.

**Comment by eneshecan** [comment]
I am not sure what is causing. Gotta test with some other account sending me messages.

**Comment by ananyo141** [comment]
Same here, using version 1.6.3

**Comment by eneshecan** [comment]
Finally, I was able to reproduce it and realized that this is the actual sound that comes from WhatsApp web which is different than the application sound we implemented. However, this seems to be a WebKit [bug](https://bugs.webkit.org/show_bug.cgi?id=189471). The reason we implemented notification sounds was that WhatsApp web notification sounds didn't work at all on Webkit. Now, it still doesn't work but it doesn't stop playing the sound once it starts to work... As a workaround for now, you can try to disable this setting from WhatsApp web settings and use the app's notification sounds.
![image](https://github.com/eneshecan/whatsapp-for-linux/assets/21038896/050f35dd-dcb7-4afb-ba69-dae05b26ec77)

**Comment by heikkiwaan** [comment]
I have the bug also. Gentoo + Guru overlay and Gnome 44.3
